### PR TITLE
Feat/url redirect composer

### PR DESCRIPTION
**Pull request by mgeerse**

### CODE REVIEW

File: functions/redirect-url-composer/1.0/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const redirectUrlComposer = async ({ url, placeholderMapping }) => {
+  placeholderMapping.map((placeholder) => {
+    url = url.replace(':' + placeholder.key, placeholder.value);
+  });
+
+  return {
+    output: url,
+  };
```
Comment (thomasvanophem) [inline functions/redirect-url-composer/1.0/index.js:8]: You could write this as
```javascript
let output = url;

placeholderMapping.forEach(({ key, value }) => {
  output = output.replace(`:${key}`, value);
});

return {
  output,
};
```

`Array.map` is used to created a new array, see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

File: config.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "host": "https://maarten-geerse-playground.bettyblocks.com",
+  "applicationId": "21c6aef175474f84a4b2815b0752b101",
+  "zone": "nl3"
+}
```
Comment (thomasvanophem) [inline config.json:5]: This file shouldn't be part of the PR

**Review by thomasvanophem — APPROVED** [review]

File: functions/redirect-url-composer/1.0/index.js (hunk 1 above)
Comment (mgeerse) [reply functions/redirect-url-composer/1.0/index.js:8]: Thanks! :)

File: config.json (hunk 2 above)
Comment (mgeerse) [reply config.json]: Removed, ty.

**Review by thomasvanophem — APPROVED** [review]
